leshan-server-demo/src/main/resources/webapp/partials/client-list.html

```diff
@@ -21,7 +21,7 @@ <h3><small><p class="text-right">Connected clients: {{clients.length}}</p></smal
 				<td>{{client.registrationDate | date:'medium'}}</td>
 				<td>{{client.lastUpdate | date:'medium'}}</td>
 				<td><i class="glyphicon glyphicon-info-sign" tooltip-html-unsafe="{{clientTooltip(client)}}"></i></td>
-				<td><span ng-style="{'visibility': client.secure ? 'visible': 'hidden'}" class="glyphicon glyphicon-lock" tooltip-html-unsafe="Communication over DTLS"></span></td>
+				<td><span ng-style="{'visibility': client.secure ? 'visible': 'hidden'}" class="glyphicon glyphicon-lock" tooltip-html-unsafe="Communication over DTLS/OSCORE"></span></td>
```

Comment:
I would keep secure = DTLS for now.
If we want to improve this we should replace the boolean by something we could show the 4 possible states (coap, coaps, coap+oscore, coaps+oscore).